ci/report-exit.js

```diff
@@ -1,6 +1,13 @@
+const { generateDiscordCloudwatchLogUrls, listECSTasks, sendDiscordNotification } = require('./helpers')
 const child_process = require('child_process')
 
-function main() {
+async function main() {
+  const taskArns = await listECSTasks()  // like  taskArns =  [ 'arn:aws:ecs:*********:************:task/ceramic-dev-tests/2466935a544f47ec9a1c3d8add235c84' ]
+  const logUrls = generateDiscordCloudwatchLogUrls(taskArns)
+  if (logUrls.length < 1) {
```

Comment:
```suggestion
  if (logUrls.length === 0) {
```

Better to keep this consistent with the way you are doing the checks in the other files

Reply:
Made consistent now.